code/datums/wires/_wires.dm

```diff
@@ -40,15 +41,16 @@
 		CRASH("Wire holder is not of the expected type!")
 
 	src.holder = holder
+	var/key = dictionary_key ? dictionary_key : holder_type
```

Comment:
```suggestion
	var/key = dictionary_key || holder_type
```
I'd consider `||` here is standard-ish and simpler.

Reply:
good idea.